sdk/documenttranslation/Azure.AI.Translator.DocumentTranslation/api/Azure.AI.Translator.DocumentTranslation.netstandard2.0.cs

```diff
@@ -94,18 +94,18 @@ public partial class DocumentTranslationOperation : Azure.AI.Translator.Document
     {
         protected DocumentTranslationOperation() { }
         public DocumentTranslationOperation(string translationId, Azure.AI.Translator.DocumentTranslation.DocumentTranslationClient client) { }
```

Comment:
we did a big shuffle of folders and namespaces for translation which is why you are seeing all the failures and conflicts. hopefully not difficult to update